src/screens/WIP/SongInfo.js

```diff
@@ -37,14 +38,41 @@ export function SongInfo() {
     };
   });
 
-  const playSong = useCallback(() => sound.play(), [sound]);
+  const playPause = () => {
+    sound.isPlaying() ? sound.pause() : sound.play();
+  };
+  const jumpPrev15Seconds = () => {
+    jumpSeconds(-10);
+  };
+  const jumpNext15Seconds = () => {
+    jumpSeconds(10);
+  };
+
+  const jumpSeconds = secondsToJump => {
+    sound.getCurrentTime(seconds => {
+      var time = seconds + secondsToJump;
```

Comment:
use `let` instead of `var` here
```suggestion
      let time = seconds + secondsToJump;
```